snowflake/tests/snowflake_connector_patch/_snowflake_connector_patch/connector.py

```diff
@@ -47,7 +47,7 @@ def execute(self, query):
             # look for ACCOUNT_NAME or CONTRACT_NUMBER since some views have
             # the same name in ACCOUNT_USAGE and ORGANIZATION_USAGE
             table_prefix = ""
-            if "ACCOUNT_NAME" or "CONTRACT_NUMBER" in query:
+            if "ACCOUNT_NAME" in query or "CONTRACT_NUMBER" in query:
```

Comment:
This looks a bit brittle. Looking a bit deeper into how the code uses the `snowflake` library, here's what I think it might make most sense: 

Since the `schema` is passed when calling `connect`
https://github.com/DataDog/integrations-core/blob/7e4e927b4e1e330e6a4191cb2d1f3aefd04b1222/snowflake/datadog_checks/snowflake/check.py#L185

How about we pass that value in our mock down to this mock `Cursor` and rely on that to know whether it's an organization query or not? I think that would match the faked behavior better.

Reply:
Sure, good idea. I updated to better handle multiple schemas